Dockerfile

```
@@ -0,0 +1,22 @@
+# To use this image, you will need to provide a GTFS file at the path pointed
+# to by the GTFS_DATA_FILE environment variable.
+
+# To build the image, make sure to npm install && npm run prepublishOnly so
+# that the dist folder is created.
+
+FROM node:12.0.0
+
+RUN mkdir -p /home/raptor/
+WORKDIR /home/raptor/
+
+COPY package*.json ./
+RUN npm install
+
+COPY dist dist
+
```

Perhaps add the below so someone building the image can just do:

docker build -t raptor-with-arg --build-arg gtfs_file=gtfs-2019-04-30.zip .


```suggestion
ARG gtfs_file="Use --build-arg gtfs_file=YOUR_GTFS_FILE.zip when building image"
COPY $gtfs_file /home/raptor/gtfs.zip
```

Unfortunately docker does not error if there no value is given so the default value is there in case no argument is passed.

---

The use of this in my mind would be someone building this image as raptor:1.0.0 or something and then anyone else who wanted to use it would create their own image something like:
```Dockerfile
FROM raptor:1.0.0

COPY gtfs-jan-2015.zip /home/raptor/gtfs.zip
```
And tagging that raptor:1.0.0-smarttis-jan-2015 or something like that.

---

Ah okay that makes sense. I think its okay then.